virttest/test_setup.py

```diff
@@ -1490,33 +1485,27 @@ def sr_iov_setup(self):
         if re_probe:
             if not self.remove_driver() or not self.modprobe_driver():
                 return False
-            if ARCH == 'ppc64le' and self.driver == 'mlx5_core':
-                set_ip = 0
-                if (self.static_ip):
-                    if (not self.start_addr_PF) or (not self.net_mask):
-                        raise exceptions.TestSetupFail(
-                            "No IP / netmask found, please populate starting IP address for PF devices in configuration file")
-                    ip_addr = utils_net.IPAddress(self.start_addr_PF)
-                    set_ip = 1
-                for PF in pf_devices:
-                    if set_ip:
-                        ip_assign = "ifconfig %s %s netmask %s up" % (
-                            PF, ip_addr, self.net_mask)
-                        logging.info("assign IP to PF device %s : %s", PF,
-                                     ip_assign)
-                        if process.system(ip_assign, shell=True,
-                                          ignore_status=True):
-                            return False
-                        ip_addr = ip_addr + 1
-                    logging.info("PF device '%s'.", PF)
-                    if not self.set_vf(PF, self.driver_option):
-                        return False
-            elif ARCH == 'x86_64':
-                pf_devices = self.get_pf_ids()
-                for PF in pf_devices:
-                    if not self.set_vf(PF, self.driver_option):
+            set_ip = 0
+            if (self.static_ip):
+                if (not self.start_addr_PF) or (not self.net_mask):
+                    raise exceptions.TestSetupFail(
+                        "No IP / netmask found, please populate starting IP address for PF devices in configuration file")
+                ip_addr = utils_net.IPAddress(self.start_addr_PF)
+                set_ip = 1
+            for PF in pf_devices:
+                if set_ip:
+                    ip_assign = "ifconfig %s %s netmask %s up" % (
+                        PF, ip_addr, self.net_mask)
```

Comment:
```
 if ARCH == 'ppc64le' and self.driver == 'mlx5_core':
     pf_devices = self.get_pf_mlx()
elif ARCH == 'x86_64':
    pf_devices = self.get_pf_ids()
```
same with the comment above, pf_device is different for  get_pf_mlx() and get_pf_ids().
for example,  we can not do "ifconfig 0000:04:00.0".